client/my-sites/exporter/index.jsx

```diff
@@ -1,62 +1,40 @@
 /**
  * External dependencies
  */
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import flowRight from 'lodash/flowRight';
 
 /**
  * Internal dependencies
  */
-import analytics from 'lib/analytics';
 import config from 'config';
-import Exporter from './exporter';
-import { getSiteSlug } from 'state/sites/selectors';
-import {
-	shouldShowProgress,
-	getSelectedPostType,
-	isExporting,
-	getExportingState,
-} from 'state/site-settings/exporter/selectors';
-import {
-	advancedSettingsFetch,
-	exportStatusFetch,
-	setPostType,
-	startExport,
-} from 'state/site-settings/exporter/actions';
-import { States } from 'state/site-settings/exporter/constants';
+import Notices from './notices';
+import ExportCard from './export-card';
+import GuidedTransferCard from './guided-transfer-card';
+import InProgressCard from './guided-transfer-card/in-progress';
 
-function mapStateToProps( state ) {
-	const siteId = state.ui.selectedSiteId;
+class Exporter extends Component {
+	render() {
+		const { isGuidedTransferInProgress } = this.props;
+		const showGuidedTransferOptions = config.isEnabled( 'manage/export/guided-transfer' );
```

Comment:
I think others might have a different opinion on this but Its my personal preference to not see config stuff inside of the render method. I'd rather see this passed down as a prop from the controller.